root.tf

```diff
@@ -31,16 +31,7 @@ module "github_sbt_assembly_log4j_plugin_repository" {
 
 module "github_preservica_config_repository" {
   source          = "git::https://github.com/nationalarchives/da-terraform-modules//github_repository_secrets"
-  repository_name = "nationalarchives/dp-preservica-config"
-  secrets = {
-    MANAGEMENT_ACCOUNT = data.aws_caller_identity.current.account_id
-    SLACK_WEBHOOK      = data.aws_ssm_parameter.github_slack_webhook.value
-  }
-}
-
-module "closure_expiration_event_lambda" {
```

Comment:
Were you meant to delete this module?

Reply:
Yeah, it's for the closure expiration stuff that we're not (currently) going to be doing.